src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -122,13 +124,15 @@ class NotebookCard extends Component {
       notebookMenu,
       title,
       div({ style: { flexGrow: 1 } }),
+      isRecent ? div({ style: { display: 'flex', color: colors.orange[0], marginRight: '2rem' } }, 'Possibly in use') : undefined,
```

Comment:
I know this has been iterated multiple times, but what about "Recently edited" instead of "Possibly in use"?

Reply:
"Recently edited" is nice because its the most clear but "Possibly in use" conveys the warning of what the recent edit implies--unsure which would be better though

Reply:
"Recently edited" is informative and "Possibly in use" is cautionary, so I think it depends on whether we want to warn users on the notebook list page or simply inform users that a notebook was recently updated. My weakly-held opinion is that we should _inform_ users on the list and then _warn_ them if they open the actual notebook. I defer to your judgement and the judgement of those you choose to listen to!

Reply:
Going with Recently Edited since it matches the actual behavior we're measuring, so it's much clearer! Ty Kate!